libs/perun/components/src/lib/members-list/members-list.component.ts

```diff
@@ -76,8 +79,11 @@ export class MembersListComponent implements OnChanges, AfterViewInit {
   setDataSource() {
     this.displayedColumns = this.displayedColumns.filter(x => !this.hideColumns.includes(x));
     if (!!this.dataSource) {
-      this.dataSource.sort = this.sort;
+      this.dataSource.filterPredicate =
+        (data: RichMember, filter: string) => parseFullName(data.user).toLowerCase().includes(filter.toLowerCase());
+      this.dataSource.filter = this.filter;
```

Comment:
the filter method is not working as it should, the filtering should work also when filtering according to id